mirdata/core.py

```diff
@@ -188,6 +188,9 @@ def default_path(self):
 
         """
         mir_datasets_dir = os.path.join(os.getenv("HOME", "/tmp"), "mir_datasets")
+        # Adapt dataset default path for TONAS
+        if self.name == "tonas":
+            self.name = "TONAS"
```

Comment:
Why do we need this? I guess the original data is in a folder called `TONAS`? For me this should just be part of the download instructions - we shouldn't put special cases for datasets within `core.py`, it should stay as generic as possible. (In short, I think these three lines should be removed)